index.bs

```diff
@@ -308,22 +308,19 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
     <h3 id="default-allowlists">Default Allowlists</h3>
     <p>Every <a>policy-controlled feature</a> has a <dfn export lt=
     "default allowlist|default allowlists">default allowlist</dfn>. The
-    <a>default allowlist</a> determines whether the feature is allowed in a
-    document with no declared policy in a top-level browsing context, and also
-    whether access to the feature is automatically delegated to documents in
-    child browsing contexts.</p>
+    <a>default allowlist</a> determines whether the feature is automatically
+    delegated to documents in child browsing contexts.</p>
     <p>The <a>default allowlist</a> for a <a
     data-lt="policy-controlled feature">feature</a> is one of these values:</p>
     <dl>
       <dt><code>*</code></dt>
-      <dd>The feature is allowed in documents in top-level browsing contexts by
-      default, and when allowed, is allowed by default to documents in child
-      browsing contexts.</dd>
+      <dd>When allowed in the top-level document, the feature is delegated by
+      default to documents in child browsing contexts.</dd>
       <dt><code>'self'</code></dt>
-      <dd>The feature is allowed in documents in top-level browsing contexts by
-      default, and when allowed, is allowed by default to same-origin domain
-      documents in child browsing contexts, but is disallowed by default in
-      cross-origin documents in child browsing contexts.</dd>
+      <dd>When allowed in the top-level document, the feature is delegated by
```

Comment:
This process applies recursively down the frame tree as well -- if the feature is allowed in some embedded frame, then it will be delegated by default to *its* same-origin children. I'd like to find a way to express that here, since it's not just about the top-level.